File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.py
def solution(n):
    # method1
    # i=1
    # while True:
    #     if (6*i)%n==0:
    #         r=i
    #         break
    #     i+=1
    # method2
    import math
    return n//math.gcd(6,n)

File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int gcd(int x,int y) {
    return x%y==0?y:gcd(y,x%y);
} 

int solution(int n) {
    int r=n/gcd(6,n);
    return r;
}